website/next.config.js

```diff
@@ -56,31 +56,7 @@ export default withNextra({
       destination: '/en/',
       permanent: true,
     },
-    {
-      source: '/en/arbitrum-faq/',
-      destination: '/en/arbitrum/arbitrum-faq/',
-      permanent: true,
-    },
-    {
-      source: '/en/querying/graph-client/',
-      destination: '/en/querying/graph-client/README/',
-      permanent: false,
-    },
-    {
-      source: '/en/developing/graph-ts/',
-      destination: '/en/developing/graph-ts/README/',
-      permanent: false,
-    },
-    {
-      source: '/en/developer/assemblyscript-api/',
-      destination: '/en/developing/graph-ts/api/',
-      permanent: true,
-    },
-    {
-      source: '/en/developing/assemblyscript-api/',
-      destination: '/en/developing/graph-ts/api/',
-      permanent: true,
-    },
```

Comment:
The only reason we ever needed to include redirects here (in addition to `nginx.conf)` is that there can still be links to the old URLs in the Markdown files. If that is the case, Next needs to be aware of them, otherwise the client-side router will link to a 404 (or maybe attempt to do a full page reload, which would be better, but still not ideal). But I hate having to maintain the same list of redirects in two places, so I went ahead and [replaced](https://github.com/graphprotocol/docs/pull/753/commits/a745368b20f96ab27eb339b46d68d76e2e13136a) all the old URLs still lingering around to the new ones, which allows us to remove these.